sklearn/metrics/pairwise.py

```diff
@@ -78,6 +78,10 @@ def check_pairwise_arrays(X, Y, precomputed=False):
         True if X is to be treated as precomputed distances to the samples in
         Y.
 
+    dtype : string, type, list of types or None (default=None)
+        Data type required for X and Y. If None, the dtype will be an
+        appropriate float type selected by _return_float_dtype.
```

Comment:
Sorry didn't spot that before: should we add a `.. versionadded:: 0.18` for the dtype parameter?